suite-native/formatters/src/components/PercentageDifferenceFormatter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Text, TextProps } from '@suite-native/atoms';
+import { useFormatters } from '@suite-common/formatters';
+
+import { FormatterProps } from '../types';
+
+type NumberValues = number;
```

Comment:
Is this type necessary?

Reply:
Not necessary, I just wanted to enforce and show, that both values need to have same type and kind of like that it's linked. This way you won't change one without the other.